### PR TITLE
Add minimatch support

### PR DESCRIPTION
**Pull request by kp101090**
Allow users to specify wild-cards in paths inputted to extension

### CODE REVIEW

File: tasks/store-publish/publishUi.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ import fs = require('fs');
 import path = require('path');
 
 import tl = require('vsts-task-lib');
+var glob = require('glob');
```
Comment (zurdev) [inline tasks/store-publish/publishUi.ts]: you should use import: import glob = require('glob')

File: tasks/store-publish/publishUi.ts [hunk 2]
```diff
@@ -112,6 +118,24 @@ function canonicalizePath(aPath: string): string
     return path.normalize(path.format(pathObj));
 }
 
+/**
+* Get appropriate files from the provided pattern
+* @param {string} path The minimatch pattern of glob to be resolved to file paths
+* @returns {string[]} file paths resolved by glob
+*/
+function resolvePathPattern(pathPattern: string) : string[] {
```
Comment (zurdev) [inline tasks/store-publish/publishUi.ts]: style: move opening brace to new line

File: tasks/store-publish/publishUi.ts [hunk 3]
```diff
@@ -112,6 +118,24 @@ function canonicalizePath(aPath: string): string
...
+    var filesList: string[] = [];
+    if (pathPattern) {
+        // Remove un-necessary quotes in path pattern, if any.
```
Comment (zurdev) [inline tasks/store-publish/publishUi.ts]: style: unnecessary is single word

**Comment by zurdev** [comment]
Looks good, minor styling comments

**Comment by elendil326** [comment]
Looks good to me.

File: tasks/store-publish/publishUi.ts (hunk 1 above)
Comment (kp101090) [inline tasks/store-publish/publishUi.ts]: Since we do not have a typings for the version of glob module we need, I will keep this!

**Pull request merged**